test/games/strategy/engine/message/RemoteMessengerTest.java

```diff
@@ -30,7 +31,7 @@
   @Before
   public void setUp() throws Exception {
     // simple set up for non networked testing
-    m_messenger = new DummyMessenger();
+    m_messenger = MockObjects.getDummyMessenger();
```

Comment:
Should not need a utility for creating mock objects.

```
IMessenger mockMessenger
```

Then create expectations in the @Test methods. Copy/pasted expectations is a bad thing, those might go into a util class if fixing the source code directly is really difficult.